index.html

```diff
@@ -186,186 +186,224 @@ <h2 class="title features__title">
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Unlimited Bandwitch</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Encrypted Connection</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">No Traffic Logs</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Works on All Devices</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Connect Anyware</p>
+              </li>
+              <li class="choose-plan__list">
+                <img
+                  class="choose-plan__check-2"
+                  src="./img/check-2.svg"
+                  width="24"
+                  height="24"
+                  alt="check-2"
+                />
+                <p class="choose-plan__text">Get New Features</p>
+              </li>
+            </ul>
+            <h3 class="choose-plan__subtitle">
+              $12 <span class="choose-plan__light">/ mo</span>
+            </h3>
+            <button class="button choose-plan__button" type="button">
+              Select
+            </button>
+          </div>
         </div>
       </section>
 
       <section class="global-net">
-        <h2 class="global-net__title">Huge Global Network of Fast VPN</h2>
-        <p class="global-net__subtitle">
+        <h2 class="global-net__title title">Huge Global Network of Fast VPN</h2>
+        <p class="global-net__subtitle subtitle">
           See <strong class="text-accent">LaslesVPN</strong> everywhere to make
-          it easier for you when you move locations.
+          it easier for you when you move <br />
+          locations.
         </p>
         <img
           class="global-net__lassmap"
```

Comment:
Smaaart 😄 But no, sorry, we need a set of images instead of one image. Here is why.
1. If someone wants to update only one logo (remove, add, or simply replace with a new picture because the client logo has been changed), they would have to update the whole big picture, it's more difficult.
2. If we want to add links to these logos so that they lead to clients websites, it's much much much easier to do it when there's a separate image for each client.
3. It's more logical, and we provide each picture with an eloquent `alt` tag saying its name.

Reply:
++